control/timeresp.py

```diff
@@ -882,6 +882,7 @@ def step_info(sysdata, T=None, T_num=None, yfinal=None,
 
             # Steady state value
             InfValue = InfValues[i, j]
+            InfValue_sign = np.sign(InfValue)
             sgnInf = np.sign(InfValue.real)
```

Comment:
?

Reply:
my bad.... in my code I use it for other thinks